bigchaindb/db/utils.py

```diff
@@ -18,54 +18,91 @@ def get_conn():
                      bigchaindb.config['database']['port'])
 
 
-def init():
-    # Try to access the keypair, throws an exception if it does not exist
-    b = bigchaindb.Bigchain()
+def get_database_name():
+    return bigchaindb.config['database']['name']
 
-    conn = get_conn()
-    dbname = bigchaindb.config['database']['name']
 
+def create_database(conn, dbname):
     if r.db_list().contains(dbname).run(conn):
         raise exceptions.DatabaseAlreadyExists('Database `{}` already exists'.format(dbname))
 
     logger.info('Create:')
```

Comment:
Yeah initially it was logging a group of messages. I wasn't sure if I should move the `log` inside the function or leave it in the `init` before the function call, as a group of messages.
I noticed now that also in the table creation and creation of the indexes functions (like: [here](https://github.com/bigchaindb/bigchaindb/pull/471/files#diff-31c0e86720248cb39d3f98060112153fR35) and [here](https://github.com/bigchaindb/bigchaindb/pull/471/files#diff-31c0e86720248cb39d3f98060112153fR62) ) it should be something like `Create %s table` or `Create index`, if we leave the `log` in the function definition.